frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.tsx

```diff
@@ -18,7 +18,7 @@ import { StyledFarmInfo, StyledSelectContainer } from './FarmInformation.style';
 
 interface SubmissionValues {
   FarmName: string;
-  Year: string;
+  Year: number;
```

Comment:
I think we should keep it as string to maintain consistency with .nmp file format, there are other instances you changed from string to number.